tasks/composer.js

```diff
@@ -20,6 +20,16 @@ module.exports = function(grunt) {
         ],
       }
     });
+    grunt.config(['composer', 'update'], {
+      options: {
+        flags: [
+          'no-interaction',
+          'no-progress',
+          'prefer-dist'
+        ],
+      }
+    });
+    grunt.config(['composer', 'drupal-scaffold'], {});
```

Comment:
I think our use of composer is now getting complicated enough that we need some more documentation on what's going on. In the pre-v1.0 branch we have a significant refactor to documentation, so it should be based on the work there.

I assume `grunt composer:drupal-scaffold` is something we might right after composer `vendor/`dependencies are installed, but before the `grunt scaffold` symlinks are in place?